docs/vars/var-operations.md

```diff
@@ -8,7 +8,6 @@ import numpy as np
 import reflex as rx
 
 from pcweb.base_state import State
-from pcweb.pages.docs.advanced_guide.custom_vars import custom_vars
 from pcweb.templates.docpage import (
```

Comment:
Not related to this pr but just noticed the [Supported Operations](http://localhost:3000/docs/vars/var-operations/#supported-operations) example is not working

Reply:
what is broken with it? It seems to work for me.